### ISSUE TITLE
Failed to compile source codes in Win 10. How to compile successfully?

### ISSUE THREAD

**Issue by hooverhe** [mybatis/generator#452]
Failed to compile 1.3.7 source codes in Win 10.  How to compile successfully?

The error is as following:

[INFO] --- license-maven-plugin:3.0:format (default) @ mybatis-generator-core ---
[INFO] Updating license headers...
[WARNING] failure occured while calling class com.mycila.maven.plugin.license.git.CopyrightRangeProvider
java.lang.RuntimeException: Could not compu[WARNING] failure occured while calling class com.mycila.maven.plugin.license.git.CopyrightRangeProvider

**Comment by jeffgbutler** [comment]
I don't get this error on Win 10, but I did get some other errors related to line endings in the Kotlin tests. I have resolved them with #453 

Please update your repo and try again.

**Comment by hooverhe** [comment]
-Dlicense.skip=true compiled successfully.

**Comment by jeffgbutler** [comment]
I'm not getting any errors on Win 10 now. My best guess is that you have a corrupted artifact in your local Maven repo.

**Comment by hooverhe** [comment]
Another question is: 
I changed the source codes to generate service and controller automatically. I also added the related Java codes. And I added the javaControllerGenerator?, javaServiceGenerator? element validation in mybatis-generator-config_1_0.dtd. After maven clean install, the javaServiceGenerator element in mybatis-generator.xml is shown as RED. Element javaServiceGenerator is not allowed here. I am not sure whether I need to change other source codes.
